Harvest.Web/ClientApp/src/Tickets/TicketWorkNotesEdit.tsx

```diff
@@ -26,7 +26,7 @@ export const TicketWorkNotesEdit = (props: Props) => {
       );
 
       if (response.ok) {
-          const data = await response.json();
+          await response.json();
```

Comment:
What is the purpose of this?  It's just taking time to do nothing.

Reply:
I pop up an alert when it is saved. I can remove that if you want

Reply:
Oh, I see what you mean...